packages/TextArea/TextArea.jsx

```diff
@@ -1,40 +1,212 @@
-import React from 'react'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
 import { componentWithName } from '@tds/util-prop-types'
 
-import FormField, { FeedbackIcon } from '@tds/shared-form-field'
+import InputFeedback from '@tds/core-input-feedback'
+import Box from '@tds/core-box'
+import Text from '@tds/core-text'
+import Paragraph from '@tds/core-paragraph'
+import { borders, position, forms } from '@tds/shared-styles'
+import {
+  colorShuttleGrey,
+  colorWhite,
+  colorPrimary,
+  colorCardinal,
+  colorAthensGrey,
+} from '@tds/core-colours'
+import { medium, mediumFont, color } from '@tds/shared-typography'
+import { FeedbackIcon } from '@tds/core-input'
+
+import safeRest from '../../shared/utils/safeRest'
```

Comment:
Please add this to `devDependencies`

Reply:
Confirmed that this is not a package in core.